src/Bundle/Maker/MakeFactory.php

```diff
@@ -202,6 +203,40 @@ private function defaultPropertiesFor(string $class, bool $allFields): iterable
         $metadata = $em->getClassMetadata($class);
         $ids = $metadata->getIdentifierFieldNames();
 
+        // TODO cleanup the code
+        // TODO class exist dont relay on fix namespaces
+        // TODO write some tests
+        // TODO test with kind of possible relations
+        // If Factory exist for related entities populate too with auto defaults
+        $relatedEntities = $metadata->associationMappings;
+        foreach ($relatedEntities as $item) {
+            // if joinColumns is not written entity is default nullable ($nullable = true;)
+            // @see vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/JoinColumn.php LINE 28
+            if (!\array_key_exists('joinColumns', $item)) {
+                continue;
+            }
+
+            // if this key is available its a ManyToMany relation but its optional, Key must not be present to be ManyToMany
+            if (\array_key_exists('joinTable', $item)) {
+                continue;
+            }
+
+            if (true === $item['joinColumns'][0]['nullable']) {
+                continue;
+            }
+
+            $joinedColumns = $item['joinColumns'];
+            $fieldName = $item['fieldName'];
+
+            $targetEntityArray = \explode('\\', $item['targetEntity']);
+            $targetEntity = \end($targetEntityArray);
+            $factory = \ucfirst($targetEntity).'Factory';
+
+            if ($this->isFactory($factory)) {
+                yield \lcfirst($fieldName) => \ucfirst($targetEntity).'Factory::createOne(),';
```

Comment:
```suggestion
                yield \lcfirst($fieldName) => \ucfirst($targetEntity).'Factory::new(),';
```

This will prevent factory's being created even when defaults are overridden.